app/src/main/java/com/openlattice/chronicle/UserIdentificationActivity.kt

```diff
@@ -20,35 +20,21 @@ class UserIdentificationActivity : AppCompatActivity() {
         setContentView(R.layout.activity_user_identification)
 
         settings = EnrollmentSettings(this)
-        saveBtn = findViewById(R.id.save_btn)
-        selectUserOptions = findViewById(R.id.select_user_options)
+        childUserBtn = findViewById(R.id.child_user_btn)
+        otherUserBtn = findViewById(R.id.other_user_btn)
 
         // listeners
-        saveBtn.setOnClickListener {
-            handleOnSave()
+        otherUserBtn.setOnClickListener {
+            handleOnSave(it.id)
```

Comment:
Is this going to automatically save when the user clicks the button and then automatically close the screen? If yes, I think the current approach is better, i.e. radio buttons along with the save button.

Reply:
IIRC the expectation with radio buttons is that their changes are effective immediately, without the requirement of a confirm button. Like toggling your wifi on your phone does not require a 'save' action. Usually a confirm is reserved for multiple checkbox style changes where they may have intermediate states.

Reply:
Yes, the changes will be automatically saved on clicking the button. Users specifically asked for that behavior.